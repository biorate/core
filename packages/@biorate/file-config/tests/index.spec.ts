import { expect } from 'chai';
import { root } from './__mocks__';

describe('@biorate/file-config', () => {
  before(async () => await root.$run());

  it('config.json', () => expect(root.config.get('base')).to.be.a('boolean').equal(true));

  it('config.debug.json', () =>
    expect(root.config.get('environment')).to.be.a('boolean').equal(true));

  it('package.json', () => expect(root.config.get('package')).toMatchSnapshot());
});
