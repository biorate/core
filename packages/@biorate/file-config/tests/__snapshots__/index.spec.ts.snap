// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@biorate/file-config package.json 1`] = `
Object {
  "author": "llevkin",
  "description": "Test package.json",
  "keywords": Array [],
  "license": "MIT",
  "name": "file-config-test",
  "private": true,
  "version": "0.0.0",
}
`;
